addons/mrp/views/mrp_production_views.xml

```diff
@@ -196,6 +197,7 @@
                     <field name="date_planned_start"/>
                     <field name="state"/>
                     <field name="activity_state"/>
+                    <field name="activity_ids"/>
```

Comment:
no need to declare it before ; it is only useful when your kanban-box template uses `t-esc="record.<fieldName>.value"` somewhere